pkg/egressgateway/manager_test.go

```diff
@@ -0,0 +1,17 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright Authors of Cilium
+
+package egressgateway
+
+import (
+	"testing"
+
+	"github.com/cilium/cilium/pkg/hive"
+)
+
+func TestCell(t *testing.T) {
```

Comment:
why is this in a separated from `manager_privileged_test.go`?

Reply:
Because it's not privileged. I can move it into the other file if you prefer.